doc/bt/ZC_CONNECT_OK.bt

```diff
@@ -25,33 +26,8 @@ byte gameMode;
 int i1;
 byte privileges;
 byte bin1[10];
-byte b1;
-byte b2;
+byte isIntegrateServer;
```

Comment:
The names should be changed in the history as well, and I guess we should talk about what we want the history to represent. So far I've only used it to document the changes to the actual packet structure, not changes in how we organize the files. The history of how the packet itself changed is important when you have older logs, for which you need to modify the template to make it compatible, but off the bat I can't think of a reason to document file changes.

Reply:
> document the changes to the actual packet structure.
This makes the most sense to me.